.github/workflows/test_cli_llama_cpp.yaml

```diff
@@ -0,0 +1,48 @@
+name: CLI Llama.cpp Tests
+
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - main
+    paths:
+      - .github/workflows/test_cli_llama_cpp.yaml
+      - "optimum_benchmark/**"
+      - "docker/**"
+      - "tests/**"
+      - "setup.py"
+  pull_request:
+    branches:
+      - main
+    paths:
+      - .github/workflows/test_cli_llama_cpp.yaml
+      - "optimum_benchmark/**"
+      - "docker/**"
+      - "tests/**"
+      - "setup.py"
+
+concurrency:
+  cancel-in-progress: true
+  group: ${{ github.workflow }}-${{ github.event.pull_request.number || github.ref }}
+
+jobs:
+  run_cli_llama_cpp_tests:
+    runs-on: ubuntu-latest
```

Comment:
I think to run on mps you'll have to use the macos-latest, if it works that is 😅.
If it doesn't (like with pytorch mps) then let's just annotate the tests as cpu tests.

Reply:
let's wait to make it MPS (due to https://github.com/actions/runner-images/issues/9918). I added them as CPU in the meantime